docs/languages/csharp/expression-trees-execution.md

```diff
@@ -0,0 +1,194 @@
+[Previous -- Interpreting Expression Trees](expression-trees-interpreting.md)
+
+# Executing Expression Trees
+An *expression tree* is a data structure that reprsents some code.
+It is not compiled and executable code. If you want to execute
+the .NET code that is represented by an expression tree, you must
+convert it into executable IL instructions. 
+## Lambda Expressions to Functions
+Not all Expression Types can be directly converted into executable
+IL. That sounds like a severe restriction, but in practice, the
```

Comment:
I'd consider rewording this sentence.  Although I think I understand that basically anything I'd use an Expression Tree for can be converted into IL, but that point does feel strong enough.  I'm left feeling like I should always be wary about if what I'm writing can be executed.